### PR TITLE
Initial conda configurations import (pixi testing on github actions)

### PR DESCRIPTION
**Pull request by j-rivero**
The PR adds the initial conda/pixi configurations together with a README explaining the directory structure and the files contained.

### CODE REVIEW

File: conda/envs/legacy/pixi.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+gflags = "*"
+glib = "*"
+gts = "*"
+jsoncpp = "*"
+libccd-double = "*"
+libprotobuf = "*"
+libsqlite = "*"
+libzip = "*"
+ogre = "1.10.12.1.*"
+ogre-next = "2.3.3.*"
```
Comment (azeey) [inline conda/envs/legacy/pixi.toml:31]: The current version in vcpkg is 2.2.x. Using 2.3.3 here will break Fortress builds. Should we not have `envs/fortress`, `envs/harmonic`,  `envs/ionic`, and `envs/jetty`?

File: conda/envs/legacy/gz-environment.yaml
Comment (azeey) [inline conda/envs/legacy/gz-environment.yaml:1]: Is this file necessary if we have the `pixi.toml` file?

File: conda/envs/legacy/pixi.toml (hunk 1 above)
Comment (j-rivero) [reply conda/envs/legacy/pixi.toml]: This first step tries to reproduce the legacy environment created by vcpkg and afterwards to create new environments for each (new?) distribution. The reason is not to alter too much the current status of the existing Windows CI builds with respect about how they are on vcpkg. I changed the version in [211a78a](https://github.com/gazebo-tooling/release-tools/pull/1192/commits/211a78aaec76c9e1aa3bd9f4b094abd539443d63)

File: conda/envs/legacy/gz-environment.yaml
Comment (j-rivero) [reply conda/envs/legacy/gz-environment.yaml:1]: It is not in use by us. It was the first file designed to work with the other mamba/micromamba/conda package managers. We could just remove it if we are not going to use it.

Comment (azeey) [reply conda/envs/legacy/gz-environment.yaml:1]: I'd rather avoid duplicating the information unless it's truly necessary.

File: conda/envs/legacy/pixi.toml (hunk 1 above)
Comment (azeey) [reply conda/envs/legacy/pixi.toml]: Okay, that makes sense. For the follow-up work, I would really like gz-rendering CI for Harmonic and Ionic to run with Ogre 2.3.3 and resolve https://github.com/gazebosim/gz-rendering/issues/745 so it would be great if we created new environments for existing distributions as well as new ones.

Comment (j-rivero) [reply conda/envs/legacy/pixi.toml]: Understood. So custom environments start with Harmonic and Ionic, maybe the same than legacy but using ogre-2.3.3 ?

Comment (azeey) [reply conda/envs/legacy/pixi.toml]: Yes, I think that's right. The only other version that might be different is CMake, but I would try with the latest first and see if we have any problems.

File: conda/envs/legacy/pixi.toml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+libwebsockets = "*"
+mesa-libegl-devel-cos7-x86_64 = "*"
+mesa-libgl-devel-cos7-x86_64 = "*"
+vulkan-headers = "*"
+
+[target.linux-aarch64.dependencies]
+libglu = "*"
+libglvnd-devel-cos7-aarch64 = "*"
+mesa-libegl-devel-cos7-aarch64 = "*"
+mesa-libgl-devel-cos7-aarch64 = "*"
```
Comment (azeey) [inline conda/envs/legacy/pixi.toml:54]: If this is mainly for windows, do we need this section?

Comment (j-rivero) [reply conda/envs/legacy/pixi.toml:54]: I don't think so. It is useful for linux users and as a long-term goal we probably want to have the platforms unified in a single source of truth so I added it here even if we are not testing/supporting it as the README mention. Hopefully it can receive patches or updates from users.

**Review by azeey — APPROVED** [review]

**Comment by azeey** [comment]
xref https://github.com/gazebo-tooling/release-tools/issues/373
